services/provider/client/client.go

```diff
@@ -133,3 +133,54 @@ func (cc *OCSProviderClient) AcknowledgeOnboarding(ctx context.Context, consumer
 
 	return cc.Client.AcknowledgeOnboarding(apiCtx, req)
 }
+
+func (cc *OCSProviderClient) ProvisionStorageClassClaim(ctx context.Context, consumerUUID, claimUUID string, storageType pb.ProvisionStorageClassClaimRequest_StorageType) (*pb.ProvisionStorageClassClaimResponse, error) {
+	if cc.Client == nil || cc.clientConn == nil {
+		return nil, fmt.Errorf("provider client is closed")
+	}
+
+	req := &pb.ProvisionStorageClassClaimRequest{
+		StorageConsumerUUID:   consumerUUID,
+		StorageClassClaimUUID: claimUUID,
```

Comment:
If we are provisioning, why do we need to send a local uid to the provider?

Reply:
Will change it to the name as we need to create the storageclassClaim CR on the provider side.